packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -42,7 +42,7 @@ BuildRequires: rubygem(gettext_i18n_rails)
 BuildRequires: rubygem(apipie-rails) >= 0.5.14
 BuildRequires: rubygem(fx) < 1.0
 BuildRequires: rubygem(pg)
-BuildRequires: rubygem(anemone)
+BuildRequires: rubygem(spidr)
```

Comment:
I think this would only work when we have a 4.13.1 gem that swtiched the dep, otherwise it won't build.

So, maybe, this commit should be part of the "4.13.1" release PR?

Reply:
Yes, you're right about that.